dtmi/azurertos/devkit/gsgmicrochipsame54-1.json

```diff
@@ -0,0 +1,73 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:azurertos:devkit:gsgmicrochipsame54;1",    
+    "@type": "Interface",
+    "displayName": "Microchip SAM E54 Getting Started Guide",
+    "description": "Example model for the Azure RTOS Microchip SAM E54 Getting Started Guide",
+    "contents": [
+        {
+            "@type": [
+                "Telemetry",
+                "Temperature"
+            ],
+            "name": "temperature",
+            "displayName": "Temperature",
+            "unit": "degreeCelsius",
+            "schema": "double"
+        },
+        {
+            "@type": [
+                "Telemetry",
+                "RelativeHumidity"
+            ],
+            "name": "humidity",
+            "displayName": "Humidity",
+            "unit": "percent",
+            "schema": "double"
+        },
+        {
+            "@type": [
+                "Telemetry",
+                "Pressure"
+            ],
+            "name": "pressure",
+            "displayName": "Pressure",
+            "unit": "kilopascal",
+            "schema": "double"
+        },
+        {
+            "@type": "Property",
+            "name": "telemetryInterval",
+            "displayName": "Telemetry Interval",
+            "description": "Specify the interval in seconds for the telemetry.",
+            "schema": "integer",
+            "writable": true
+        },
+        {
+            "@type": "Property",
+            "name": "ledState",
+            "displayName": "LED state",
+            "description": "Returns the current state of the onboard LED.",
+            "schema": "boolean"
+        },
+        {
+            "@type": "Command",
```

Comment:
consider add an empty response

Reply:
How do you add an empty response? dmr-client complained when I tried this.